vm/devices/storage/disk_nvme/nvme_driver/src/namespace.rs

```diff
@@ -579,3 +664,15 @@ fn nvm_cmd(opcode: nvm::NvmOpcode, nsid: u32) -> spec::Command {
         ..FromZeroes::new_zeroed()
     }
 }
+
+/// Save/restore NVMe namespace data.
+#[derive(Protobuf, Clone, Debug)]
+#[mesh(package = "underhill")]
+pub struct SavedNamespaceData {
+    #[mesh(1)]
+    pub nsid: u32,
+    #[mesh(2)]
+    pub block_count: u64,
+    #[mesh(3)]
+    pub identify_ns: [u8; 4096],
```

Comment:
```suggestion
    #[mesh(encoding = "mesh::payload::encoding::ZeroCopyEncoding")]
    pub identify_ns: nvme_spec::nvm::IdentifyNamespace,
```

Reply:
Thanks for pointing to ZeroCopyEncoding. I didn't know it can be used for this purpose as there are no examples in the code...